src/server/routes/register.ts

```diff
@@ -114,7 +132,7 @@ router.post(
 
     trackMetric(Metrics.REGISTER_SUCCESS);
 
-    return res.redirect(303, returnUrl);
+    return res.redirect(303, Routes.REGISTRATION_EMAIL_SENT);
```

Comment:
We want to redirect to the email sent page after a successful registration, not the return url (that will happen later after the welcome flow is complete)